dom-querying/main.js

```diff
@@ -0,0 +1,18 @@
+
+var $heading = document.querySelector('h1');
+console.log($heading);
```

Comment:
make sure you are adding descriptions to your console logs so you know what you're logging out!

`console.log('$heading', $heading);`